compose/reference/pull.md

```diff
@@ -13,4 +13,45 @@ Options:
     --parallel              Pull multiple images in parallel.
 ```
 
-Pulls service images.
+Pulls an image associated with a service defined in a `docker-compose.yml` or `docker-stack.yml` file, but does not start containers based on those images.
```

Comment:
@shin- it would be cool to update `docker-compose pull --help` to also include this nugget of information